src/components/Pagination.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import "./pagination.scss";
+interface pageAndLimit {
+  page: number;
+  limit: number;
+}
+
+export const Pagination = () => {
+  const [query, setQuery] = React.useState<string>("");
+  const [pagin, setPagin] = React.useState<pageAndLimit>({ page: 1, limit: 2 });
+  const [products, setProducts] = React.useState<any[]>();
```

Comment:
try to minimize the usage of any
I probably understand why used any
In the future, you don't have to add all properties to the interface you can add only those you need
Example
in response, I receive an extremely big object but I use only 5 fields from that object
so in interface, I add only those 5 fields